qemu/tests/nic_hotplug.py

```diff
@@ -36,7 +36,8 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters.
     :param env:    Dictionary with test environment.
     """
-    def renew_ip_address(session, mac, is_linux_guest=True):
+    def renew_ip_address(mac, is_linux_guest=True):
+        session = vm.wait_for_serial_login(timeout=login_timeout)
```

Comment:
hi, why use serial login here

Reply:
1. this is a nic hot plug test case. so we'd better not to use nic login
2. this will ensure that the serial session is used in single instance at same time.